prisma/schema.prisma

```diff
@@ -45,3 +46,24 @@ model Trainer {
   gender        String
   classes       Class[]
 }
+
+model Profile {
+  id        Int      @id @default(autoincrement())
+  picture   String   @db.VarChar(255)
+  firstName String   @db.VarChar(255)
+  lastname  String   @db.VarChar(255) 
+  member    Member   @relation(fields: [memberId], references: [id])
+  memberId  Int
+  address   Address?
+}
+
+model Address { 
+id          Int     @id @default(autoincrement())
+houseNumber	Int
```

Comment:
It doesn't matter for these projects, but this may need to be a string in a real world project to cover addresses that have house numbers with letters in like "12A". Just something to keep in mind!

Reply:
Understood, Will be changing this according as we go forward
thanks fo the feedback